[2025-01-26T01:01:42.400-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T01:01:42.423-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: InsertIntoElastic.inject_data_into_es scheduled__2021-03-11T00:00:00+00:00 [queued]>
[2025-01-26T01:01:42.440-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: InsertIntoElastic.inject_data_into_es scheduled__2021-03-11T00:00:00+00:00 [queued]>
[2025-01-26T01:01:42.441-0300] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-01-26T01:01:42.477-0300] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): inject_data_into_es> on 2021-03-11 00:00:00+00:00
[2025-01-26T01:01:42.492-0300] {standard_task_runner.py:72} INFO - Started process 120406 to run task
[2025-01-26T01:01:42.495-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'InsertIntoElastic', 'inject_data_into_es', 'scheduled__2021-03-11T00:00:00+00:00', '--job-id', '4607', '--raw', '--subdir', 'DAGS_FOLDER/DAGInjectElastic.py', '--cfg-path', '/tmp/tmpdxvanm8t']
[2025-01-26T01:01:42.497-0300] {standard_task_runner.py:105} INFO - Job 4607: Subtask inject_data_into_es
[2025-01-26T01:01:42.564-0300] {task_command.py:467} INFO - Running <TaskInstance: InsertIntoElastic.inject_data_into_es scheduled__2021-03-11T00:00:00+00:00 [running]> on host lucas-Aspire-A514-54
[2025-01-26T01:01:42.788-0300] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='lucas' AIRFLOW_CTX_DAG_ID='InsertIntoElastic' AIRFLOW_CTX_TASK_ID='inject_data_into_es' AIRFLOW_CTX_EXECUTION_DATE='2021-03-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-11T00:00:00+00:00'
[2025-01-26T01:01:42.789-0300] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T01:01:43.423-0300] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow/dags/DAGInjectElastic.py", line 130, in inject_data_into_es
    result = helpers.bulk(es_client, df_custom)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/helpers/actions.py", line 540, in bulk
    for ok, item in streaming_bulk(
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/helpers/actions.py", line 435, in streaming_bulk
    for bulk_data, bulk_actions in _chunk_actions(
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/helpers/actions.py", line 234, in _chunk_actions
    for action, data in actions:
  File "/home/lucas/airflow/dags/DAGInjectElastic.py", line 10, in generator
    for c, line in df2.iterrows():
                   ^^^^^^^^^^^^
AttributeError: 'dict' object has no attribute 'iterrows'
[2025-01-26T01:01:43.466-0300] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=InsertIntoElastic, task_id=inject_data_into_es, run_id=scheduled__2021-03-11T00:00:00+00:00, execution_date=20210311T000000, start_date=20250126T040142, end_date=20250126T040143
[2025-01-26T01:01:43.499-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T01:01:43.499-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 4607 for task inject_data_into_es ('dict' object has no attribute 'iterrows'; 120406)
Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow/dags/DAGInjectElastic.py", line 130, in inject_data_into_es
    result = helpers.bulk(es_client, df_custom)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/helpers/actions.py", line 540, in bulk
    for ok, item in streaming_bulk(
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/helpers/actions.py", line 435, in streaming_bulk
    for bulk_data, bulk_actions in _chunk_actions(
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/helpers/actions.py", line 234, in _chunk_actions
    for action, data in actions:
  File "/home/lucas/airflow/dags/DAGInjectElastic.py", line 10, in generator
    for c, line in df2.iterrows():
                   ^^^^^^^^^^^^
AttributeError: 'dict' object has no attribute 'iterrows'
[2025-01-26T01:01:43.551-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-01-26T01:01:43.564-0300] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-26T01:01:43.564-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
